SCClassLibrary/Common/Collections/Dictionary.sc

```diff
@@ -296,12 +296,17 @@ Dictionary : Set {
 
 	isAssociationArray { ^false }
 
-	asPairs {
-		^this.getPairs
+	asPairs { |class|
+		var res = (class ? Array).new(this.size * 2);
+		this.pairsDo { |key, val|
+			res.add(key).add(val);
+		};
+		^res
 	}
 
-	asDict {
-		^this
+	asDict { arg mergeFunc, class;
```

Comment:
I'd only do this for the whole file. I'd prefer to keep the surrounding style, and change it some time in the future, all at once in a separate PR.